> @al-niessner - deployed, executed and received the below error in `_stderr`:
> 
> ```
> Traceback (most recent call last):
>   File "/home/ops/verdi/ops/coseismic_enumerator/iterate.py", line 80, in <module>
>     if __name__ == '__main__': main()
>   File "/home/ops/verdi/ops/coseismic_enumerator/iterate.py", line 57, in main
>     try: active.process (aoi)
>   File "/home/ops/verdi/ops/coseismic_enumerator/active.py", line 117, in process
>     track_number=aoi['metadata']['context']['track_number'])
> KeyError: 'track_number'
> ```

@marjo-luc the quoted trace back above in this comment means that the track number has moved in the AOI. Interestingly it moved since the test run after including the water mask and now. Was the move intentional or is the move an error or is the enumerator not using the stable location of the track number? Please let me know.

@riverma if the _stdout.txt is still around, can we grab it?

--- a/es/__init__.py
+++ b/es/__init__.py
@@ -61,7 +61,7 @@ def query (request:{}, index:str='',
     total = None
     while total is None or len(result) < total:
         content['from'] = len(result) + es_from
-        response = requests.post (grq_url, data=json.dumps(content))
+        response = requests.get (grq_url, data=json.dumps(content))
         response.raise_for_status()
         data = json.loads (response.text, encoding='ascii')
         result.extend (data.get('hits', {}).get('hits', []))

